pkg/cmd/run.go

```diff
@@ -33,19 +33,31 @@ func NewRunMicroshiftCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "run",
 		Short: "Run MicroShift",
-		RunE: func(cmd *cobra.Command, args []string) error {
-			cfg, err := config.ActiveConfig()
-			if err != nil {
-				return err
-			}
-			// Things to very badly if the node's name has changed
-			// since the last time the server started.
-			err = cfg.EnsureNodeNameHasNotChanged()
-			if err != nil {
-				return err
-			}
-			return RunMicroshift(cfg)
-		},
+	}
+
+	var master bool
+
+	flags := cmd.Flags()
+	flags.BoolVar(&master, "master", false, "act as control plane node in multinode mode")
```

Comment:
Using a command line flag means the instructions for running in the configuration for certification won't be able to use the default systemd unit. That's not necessarily a bad thing. For example, it means it will be easier to not support multi-node mode. But it's something we will need to be aware of and handle in our automation for the certification test job.

Reply:
@dhellmann understood, good point! 
I could change it to a hidden config file option instead if that sounds better.
I'll keep it for now, and change it anytime later if you prefer.

Reply:
Yes, let's leave it as a command line option for now. It will be easy to change it later to make the automation more convenient, if we need to.